.storybook/themes/news.scss

```diff
@@ -1992,6 +2084,34 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				search-results-list-search-bar: (
```

Comment:
Do not see this in the Sass file

Reply:
yep, leftover from my rework of the tokens